packages/forma-36-react-components/src/components/Pill/Pill.tsx

```diff
@@ -56,6 +56,7 @@ export class Pill extends Component<PillProps> {
         {onClose && (
           <button
             type="button"
+            aria-label="button"
```

Comment:
`aria-label` must describe what the button does, which I believe is "close" in this case, right?